src/components/Text/Text.tsx

```diff
@@ -50,52 +53,32 @@ export interface TextProps extends NestableBaseComponentProps {
  */
 export default class Text extends React.Component<TextProps> {
   public render() {
+    return <BlockContext.Consumer>{blockContext => this.getContent(blockContext.textSize)}</BlockContext.Consumer>;
```

Comment:
was there a reason to move the content into getContent? maybe it's just all the red and green lines, but would it be easier to understand what was being rendered if the markup was right here

Reply:
Yeah, it was to work around the `jsx-wrap-multiline` tslint rule. Am I missing a way to do this within that constraint? Is this a rule we want to keep?

Reply:
Sorry, wrong ts-lint rule, it's this one:
```
[tslint] Multiline JS expressions inside JSX are forbidden (jsx-no-multiline-js)
```

Reply:
ah. we've removed that linting rule in MC since libraries are using more render props these days, we could remove it here but don't worry about it here